src/plugins/region_map/public/components/map_choice_options.test.tsx

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as React from 'react';
+import { MapChoiceOptions } from './map_choice_options';
+import { screen, render } from '@testing-library/react';
+import { fireEvent, getByTestId } from '@testing-library/dom';
+import '@testing-library/jest-dom';
+
+describe('map_choice_options', () => {
+  it('renders the MapChoiceOptions based on the props provided', async () => {
+    const props = jest.mock;
+    const vis = {
```

Comment:
nit: since most of the fields here remain the same across the tests, you can set a default one and only override the props you need to be different for each test so that reading the tests is a lot easier.

e.g. 
```js
const DEFAULT = {
      colorSchema: {},
      outlineWeight: {},
      ...
}

// And in test 1

const viz = {
    ...DEFAULT,
    ...{
        customProp: something
    }
}
```

The same comment goes for the other tests too.